### DISCUSSION TITLE
How to set window size on startup

### DISCUSSION THREAD

**Discussion by jkj** [akiyosi/goneovim#507] [General]
I absolutely love this program. The one thing I can't seem to get right is sizing the screen correctly on startup. --geometry works but it takes pixels as units when rows and columns would be more appropriate (the way xterm does). If I use `:set lines=55` and `:set columns=225` once its started, it sizes to the desired size just fine but I cannot seem to get it to do that on startup. Please can someone help me. I am using neovim 1.10 nightly and whatever the latest commit of goneovim is. Thank you in advance for any help.

**Comment by gerroon** [comment]
I have a similar issue in Windows. It always opens very small :( It would be so great if there is a way to set a window size based on the current window size or make it remember the last size.

**Comment by akiyosi** [comment]
@jkj 
Sorry for not replying for a long time.
I have added support for specifying geometry by number of rows and columns from the command line in the latest XXX. Please try it.|
https://github.com/akiyosi/goneovim/actions/runs/8188882606

```
/path/to/goneovim -- --cmd 'set lines=30 columns=50'            
```

@gerroon 
To remember the window size at the last startup, set the following in settings.toml

```
[Editor]
RestoreWindowGeometry = true
```

In addition, height and width can also be specified from the settings.

```
[Editor]
Width = 800
Height = 600
```
